python/tvm/micro/build.py

```diff
@@ -74,8 +74,8 @@ def path(self):
 
 _CRT_DEFAULT_OPTIONS = {
     "cflags": ["-std=c11"] + _COMMON_CFLAGS,
-    "ccflags": ["-std=c++11"] + _COMMON_CFLAGS,
-    "ldflags": ["-std=c++11"],
+    "ccflags": ["-std=gnu++11"] + _COMMON_CFLAGS,
```

Comment:
I think current implementation of linked-params uses hexadecimal notation that requires gnu++. This is partly because export_library saves c-source modules .cc.

Reply:
Alright, I think the correct thing to do is make export library save the runtime modules as .c not .cc. Thus, did that and reverted these changes

Reply:
ah, I just realized that was what was going on here. I agree we shouldn't be generating c++ for the c backend.